### ISSUE TITLE
1 visitor-multiple session

### ISSUE THREAD

**Issue by centoasa** [PrestaShop/PrestaShop#12246]
prestashop 1.7.5.0
Debian 4.9.110-3+deb9u2  x86_64
php 7.2.14
MySQL: 10.1.37-MariaDB-0+deb9u1
as you can see on this document https://1drv.ms/t/s!AnkbspQIR_C66z8seujS3BVYK89d for every visitor (IP) prestashop have a greta number of session with the same IP
It's normal? or it's an issue?

**Comment by khouloudbelguith** [comment]
Hi @centoasa,

Could you please add steps to reproduce the issue in a new comment.
Thanks!

**Comment by centoasa** [comment]
backoffice-->https://www.100asa.it/admin/index.php?controller=AdminStats&token=ee73eaa2d82bb861c97af7fb56332ddf&module=statslive and then see all visitors in the attached file first

**Comment by khouloudbelguith** [comment]
@centoasa,

I have the same issue but in other cases.
The address IP for a different guests:
![image](https://user-images.githubusercontent.com/16067358/51488871-0577ec00-1da7-11e9-8e4d-2990865bdfcf.png)
I use the same computer but I use different browsers for the same shop(Chrome, navigate browser chrome, FireFox, Safari,..)
That's why, I have the same IP address.
I think it is a normal behavior.
Thanks!

**Comment by khouloudbelguith** [comment]
@centoasa, have you the same conditions described by me?
Thanks!

**Comment by centoasa** [comment]
@khouloudbelguith no, it's not my case. By tawk chat, I monitor visitors, and in the same time, same customer have 1 only session but a large amount of ip session on presta-backoffice

**Comment by centoasa** [comment]
do you think this as a little strange? https://nimb.ws/8TsHEm

**Comment by khouloudbelguith** [comment]
@centoasa, , they are robots? did you generate the robots.txt file?

Generating the robots.txt file: A robots.txt file allows you to block robots and other automated software that browse the Web to find more pages to add to their company's server. You will want to give full access to some robots, like those of Google or Yahoo !, while others will have to find closed doors, such as spam robots, content thieves, e-mail address collectors, etc.

**Comment by centoasa** [comment]
yes, I already have the robot file

**Comment by centoasa** [comment]
now are 333... please see https://1drv.ms/t/s!AnkbspQIR_C660R5cBuLqhvqkFQ0

**Comment by BenTen** [comment]
I'm not an expert, but the number 333 seems a bit ridiculous for a Ddos attack.

**Comment by khouloudbelguith** [comment]
@centoasa, We need to retrieve the PHP error log and the debug mode report in order to find out what's wrong.
It could be due to a proxy server.
You can use PrestaShop Geo Location Feature. It is really very good to block the IP.
Thanks!

**Comment by centoasa** [comment]
@khouloudbelguith maybe nginx proxy?
sorry: what's "PrestaShop Geo Location Feature" ?

**Comment by khouloudbelguith** [comment]
@centoasa, you are welcome!
Thanks!

**Comment by centoasa** [comment]
@khouloudbelguith  Geo Location Feature-->not solve the issue
as you can see here  https://nimb.ws/w9sR7Q this ip is normal customer, that open open page (pigment compa epson), but prestashop record it as multiple istance https://nimb.ws/o8HymT on the same time!

**Comment by khouloudbelguith** [comment]
@centoasa, thanks for your feedback.
I manage to reproduce the issue with PS1.7.5.0 & ps1.7.4.4.
When I'm trying to navigate(as a guest) on the FO(different pages: home page, product page,..)
In the BO, PrestaShop records it as multiple instances at the same time.
![image](https://user-images.githubusercontent.com/16067358/51544368-09b41000-1e60-11e9-80e5-ab214e9dd6df.png)
https://drive.google.com/file/d/1BC28OXErZINi-hnGyWfluz61RPD4Q4Am/view
We will see how to fix it.
Thanks!

**Comment by centoasa** [comment]
wow! great! I'm not lonely! 👍
